{"uid":"beb1eebcccd68495","name":"test_dynamic_steps","fullName":"tests.test_steps#test_dynamic_steps","historyId":"7c7c5e5f20fe2663d33f122f5ce2ca5a","time":{"start":1741385570076,"stop":1741385570277,"duration":201},"status":"broken","statusMessage":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created: probably user data directory is already in use, please specify a unique value for --user-data-dir argument, or don't use --user-data-dir\nStacktrace:\n#0 0x55a516fbf14a <unknown>\n#1 0x55a516a5cb80 <unknown>\n#2 0x55a516a9603f <unknown>\n#3 0x55a516a91d7f <unknown>\n#4 0x55a516ae2819 <unknown>\n#5 0x55a516ae1d46 <unknown>\n#6 0x55a516ad3ca3 <unknown>\n#7 0x55a516a9ff08 <unknown>\n#8 0x55a516aa1071 <unknown>\n#9 0x55a516f88b5b <unknown>\n#10 0x55a516f8cae2 <unknown>\n#11 0x55a516f74967 <unknown>\n#12 0x55a516f8d6d4 <unknown>\n#13 0x55a516f58c7f <unknown>\n#14 0x55a516fadcd8 <unknown>\n#15 0x55a516fadea9 <unknown>\n#16 0x55a516fbdfc6 <unknown>\n#17 0x7f0453a9caa4 <unknown>\n#18 0x7f0453b29c3c <unknown>","statusTrace":"def test_dynamic_steps():\n        with allure.step('Открываем главную страницу'):\n            browser.config.window_height = 1080  # высота браузера\n            browser.config.window_width = 1920 # ширина браузера\n>           browser.open('https://github.com/')\n\ntests/test_steps.py:10: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selene/core/_browser.py:93: in open\n    self.config._executor.get_url(relative_or_absolute_url)\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selene/core/configuration.py:263: in get_url\n    self.config._driver_get_url_strategy(self.config)(url)\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selene/core/configuration.py:178: in get\n    driver = config.driver\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selene/core/configuration.py:305: in __get__\n    driver = config.build_driver_strategy(config)\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selene/core/configuration.py:129: in _build_local_driver_by_name_or_remote_by_url_and_options\n    return {  # type: ignore\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selene/core/configuration.py:66: in build_chrome\n    return Chrome(\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/chrome/webdriver.py:45: in __init__\n    super().__init__(\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/chromium/webdriver.py:66: in __init__\n    super().__init__(command_executor=executor, options=options)\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:250: in __init__\n    self.start_session(capabilities)\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:342: in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:429: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f1552a15d10>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"session not created\",\"message\":\"session not created: probably user data d...\\\\n#16 0x55a516fbdfc6 \\\\u003Cunknown>\\\\n#17 0x7f0453a9caa4 \\\\u003Cunknown>\\\\n#18 0x7f0453b29c3c \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.SessionNotCreatedException: Message: session not created: probably user data directory is already in use, please specify a unique value for --user-data-dir argument, or don't use --user-data-dir\nE       Stacktrace:\nE       #0 0x55a516fbf14a <unknown>\nE       #1 0x55a516a5cb80 <unknown>\nE       #2 0x55a516a9603f <unknown>\nE       #3 0x55a516a91d7f <unknown>\nE       #4 0x55a516ae2819 <unknown>\nE       #5 0x55a516ae1d46 <unknown>\nE       #6 0x55a516ad3ca3 <unknown>\nE       #7 0x55a516a9ff08 <unknown>\nE       #8 0x55a516aa1071 <unknown>\nE       #9 0x55a516f88b5b <unknown>\nE       #10 0x55a516f8cae2 <unknown>\nE       #11 0x55a516f74967 <unknown>\nE       #12 0x55a516f8d6d4 <unknown>\nE       #13 0x55a516f58c7f <unknown>\nE       #14 0x55a516fadcd8 <unknown>\nE       #15 0x55a516fadea9 <unknown>\nE       #16 0x55a516fbdfc6 <unknown>\nE       #17 0x7f0453a9caa4 <unknown>\nE       #18 0x7f0453b29c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py:232: SessionNotCreatedException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created: probably user data directory is already in use, please specify a unique value for --user-data-dir argument, or don't use --user-data-dir\nStacktrace:\n#0 0x55a516fbf14a <unknown>\n#1 0x55a516a5cb80 <unknown>\n#2 0x55a516a9603f <unknown>\n#3 0x55a516a91d7f <unknown>\n#4 0x55a516ae2819 <unknown>\n#5 0x55a516ae1d46 <unknown>\n#6 0x55a516ad3ca3 <unknown>\n#7 0x55a516a9ff08 <unknown>\n#8 0x55a516aa1071 <unknown>\n#9 0x55a516f88b5b <unknown>\n#10 0x55a516f8cae2 <unknown>\n#11 0x55a516f74967 <unknown>\n#12 0x55a516f8d6d4 <unknown>\n#13 0x55a516f58c7f <unknown>\n#14 0x55a516fadcd8 <unknown>\n#15 0x55a516fadea9 <unknown>\n#16 0x55a516fbdfc6 <unknown>\n#17 0x7f0453a9caa4 <unknown>\n#18 0x7f0453b29c3c <unknown>","statusTrace":"def test_dynamic_steps():\n        with allure.step('Открываем главную страницу'):\n            browser.config.window_height = 1080  # высота браузера\n            browser.config.window_width = 1920 # ширина браузера\n>           browser.open('https://github.com/')\n\ntests/test_steps.py:10: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selene/core/_browser.py:93: in open\n    self.config._executor.get_url(relative_or_absolute_url)\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selene/core/configuration.py:263: in get_url\n    self.config._driver_get_url_strategy(self.config)(url)\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selene/core/configuration.py:178: in get\n    driver = config.driver\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selene/core/configuration.py:305: in __get__\n    driver = config.build_driver_strategy(config)\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selene/core/configuration.py:129: in _build_local_driver_by_name_or_remote_by_url_and_options\n    return {  # type: ignore\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selene/core/configuration.py:66: in build_chrome\n    return Chrome(\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/chrome/webdriver.py:45: in __init__\n    super().__init__(\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/chromium/webdriver.py:66: in __init__\n    super().__init__(command_executor=executor, options=options)\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:250: in __init__\n    self.start_session(capabilities)\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:342: in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:429: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f1552a15d10>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"session not created\",\"message\":\"session not created: probably user data d...\\\\n#16 0x55a516fbdfc6 \\\\u003Cunknown>\\\\n#17 0x7f0453a9caa4 \\\\u003Cunknown>\\\\n#18 0x7f0453b29c3c \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.SessionNotCreatedException: Message: session not created: probably user data directory is already in use, please specify a unique value for --user-data-dir argument, or don't use --user-data-dir\nE       Stacktrace:\nE       #0 0x55a516fbf14a <unknown>\nE       #1 0x55a516a5cb80 <unknown>\nE       #2 0x55a516a9603f <unknown>\nE       #3 0x55a516a91d7f <unknown>\nE       #4 0x55a516ae2819 <unknown>\nE       #5 0x55a516ae1d46 <unknown>\nE       #6 0x55a516ad3ca3 <unknown>\nE       #7 0x55a516a9ff08 <unknown>\nE       #8 0x55a516aa1071 <unknown>\nE       #9 0x55a516f88b5b <unknown>\nE       #10 0x55a516f8cae2 <unknown>\nE       #11 0x55a516f74967 <unknown>\nE       #12 0x55a516f8d6d4 <unknown>\nE       #13 0x55a516f58c7f <unknown>\nE       #14 0x55a516fadcd8 <unknown>\nE       #15 0x55a516fadea9 <unknown>\nE       #16 0x55a516fbdfc6 <unknown>\nE       #17 0x7f0453a9caa4 <unknown>\nE       #18 0x7f0453b29c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py:232: SessionNotCreatedException","steps":[{"name":"Открываем главную страницу","time":{"start":1741385570076,"stop":1741385570275,"duration":199},"status":"broken","statusMessage":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created: probably user data directory is already in use, please specify a unique value for --user-data-dir argument, or don't use --user-data-dir\nStacktrace:\n#0 0x55a516fbf14a <unknown>\n#1 0x55a516a5cb80 <unknown>\n#2 0x55a516a9603f <unknown>\n#3 0x55a516a91d7f <unknown>\n#4 0x55a516ae2819 <unknown>\n#5 0x55a516ae1d46 <unknown>\n#6 0x55a516ad3ca3 <unknown>\n#7 0x55a516a9ff08 <unknown>\n#8 0x55a516aa1071 <unknown>\n#9 0x55a516f88b5b <unknown>\n#10 0x55a516f8cae2 <unknown>\n#11 0x55a516f74967 <unknown>\n#12 0x55a516f8d6d4 <unknown>\n#13 0x55a516f58c7f <unknown>\n#14 0x55a516fadcd8 <unknown>\n#15 0x55a516fadea9 <unknown>\n#16 0x55a516fbdfc6 <unknown>\n#17 0x7f0453a9caa4 <unknown>\n#18 0x7f0453b29c3c <unknown>\n\n","statusTrace":"  File \"/home/runner/work/homework_10_allure/homework_10_allure/tests/test_steps.py\", line 10, in test_dynamic_steps\n    browser.open('https://github.com/')\n    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selene/core/_browser.py\", line 93, in open\n    self.config._executor.get_url(relative_or_absolute_url)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selene/core/configuration.py\", line 263, in get_url\n    self.config._driver_get_url_strategy(self.config)(url)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selene/core/configuration.py\", line 178, in get\n    driver = config.driver\n             ^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selene/core/configuration.py\", line 305, in __get__\n    driver = config.build_driver_strategy(config)\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selene/core/configuration.py\", line 129, in _build_local_driver_by_name_or_remote_by_url_and_options\n    return {  # type: ignore\n           ~~~~~~~~~~~~~~~~~\n    ...<32 lines>...\n        )\n        ~\n    )()\n    ~^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selene/core/configuration.py\", line 66, in build_chrome\n    return Chrome(\n        service=config.driver_service or ChromeService(),\n        options=config.driver_options,\n    )\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/chrome/webdriver.py\", line 45, in __init__\n    super().__init__(\n    ~~~~~~~~~~~~~~~~^\n        browser_name=DesiredCapabilities.CHROME[\"browserName\"],\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    ...<3 lines>...\n        keep_alive=keep_alive,\n        ^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/chromium/webdriver.py\", line 66, in __init__\n    super().__init__(command_executor=executor, options=options)\n    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 250, in __init__\n    self.start_session(capabilities)\n    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 342, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n               ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 429, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_steps"},{"name":"host","value":"fv-az1377-716"},{"name":"thread","value":"2342-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_steps"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":3,"skipped":0,"passed":0,"unknown":0,"total":3},"items":[{"uid":"41bfa652feedd15d","reportUrl":"https://hiller994.github.io/homework_10_allure/6//#testresult/41bfa652feedd15d","status":"broken","statusDetails":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created: probably user data directory is already in use, please specify a unique value for --user-data-dir argument, or don't use --user-data-dir\nStacktrace:\n#0 0x5637982f314a <unknown>\n#1 0x563797d90b80 <unknown>\n#2 0x563797dca03f <unknown>\n#3 0x563797dc5d7f <unknown>\n#4 0x563797e16819 <unknown>\n#5 0x563797e15d46 <unknown>\n#6 0x563797e07ca3 <unknown>\n#7 0x563797dd3f08 <unknown>\n#8 0x563797dd5071 <unknown>\n#9 0x5637982bcb5b <unknown>\n#10 0x5637982c0ae2 <unknown>\n#11 0x5637982a8967 <unknown>\n#12 0x5637982c16d4 <unknown>\n#13 0x56379828cc7f <unknown>\n#14 0x5637982e1cd8 <unknown>\n#15 0x5637982e1ea9 <unknown>\n#16 0x5637982f1fc6 <unknown>\n#17 0x7f03e589caa4 <unknown>\n#18 0x7f03e5929c3c <unknown>","time":{"start":1741385275152,"stop":1741385275357,"duration":205}},{"uid":"8f198c364d8ca12","reportUrl":"https://hiller994.github.io/homework_10_allure/5//#testresult/8f198c364d8ca12","status":"broken","statusDetails":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created: probably user data directory is already in use, please specify a unique value for --user-data-dir argument, or don't use --user-data-dir\nStacktrace:\n#0 0x561d6aa8714a <unknown>\n#1 0x561d6a524b80 <unknown>\n#2 0x561d6a55e03f <unknown>\n#3 0x561d6a559d7f <unknown>\n#4 0x561d6a5aa819 <unknown>\n#5 0x561d6a5a9d46 <unknown>\n#6 0x561d6a59bca3 <unknown>\n#7 0x561d6a567f08 <unknown>\n#8 0x561d6a569071 <unknown>\n#9 0x561d6aa50b5b <unknown>\n#10 0x561d6aa54ae2 <unknown>\n#11 0x561d6aa3c967 <unknown>\n#12 0x561d6aa556d4 <unknown>\n#13 0x561d6aa20c7f <unknown>\n#14 0x561d6aa75cd8 <unknown>\n#15 0x561d6aa75ea9 <unknown>\n#16 0x561d6aa85fc6 <unknown>\n#17 0x7fb288e9caa4 <unknown>\n#18 0x7fb288f29c3c <unknown>","time":{"start":1741384536411,"stop":1741384536615,"duration":204}}]},"tags":[]},"source":"beb1eebcccd68495.json","parameterValues":[]}